### PR TITLE
Lesson 3 Steps

### PR DESCRIPTION
**Pull request by nildo**
Added the steps for lesson 3. Also, added a support to make the robot initially holding a gear. I'm still making the new sprites.

### CODE REVIEW

File: src/lesson/lesson03.js [hunk 1]
```diff
@@ -518,46 +522,852 @@ Lesson03ExerciseStepPlayer.prototype = {
...
+        O robô precisa chegar em frente à máquina e colocar a engrenagem.
+      </p>,
+      <div>
+        <p>
+          Muito bem, o robô pegou uma engrenagem, e está segurando
+          ela. Agora, ele precisa colocá-la em uma máquina que
+          está com uma engrenagem faltando.
+          Para isso, ele precisa chegar em frente à máquina e utilizar o seu
+          comando “<b>P</b>” (do inglês <i>Put</i>, que significa
+          <b>colocar</b>).
```
Comment (gpoesia) [inline src/lesson/lesson03.js]: By default, there's no spacing around tags when they start in a new line, so the text reads "significacolocar" (with the "colocar" in bold). You can put an explicit space: add &nbsp<b></b>; before <b><</b>b>colocar<b><</b>/b>

File: src/lesson/lesson03.js [hunk 2]
```diff
@@ -518,46 +522,852 @@ Lesson03ExerciseStepPlayer.prototype = {
...
+    new Lesson.LessonStep(
+      <p>
+        Você vai precisar verificar se a engrenagem está em dois
+        locais diferentes.
+      </p>,
+      <div>
+        <p>
+          Muito bem! Encontramos uma engrenagem na área escura
+          da sala anterior. Para isso, você utilizou
+          <b>{"eng?{G}"}</b>, que só executa o comando G se a
```
Comment (gpoesia) [inline src/lesson/lesson03.js]: add &nbsp<b></b>; before the <b> tag.

File: src/lesson/lesson03.js [hunk 3]
```diff
@@ -518,46 +522,852 @@ Lesson03ExerciseStepPlayer.prototype = {
...
+          new Grid.Position(3, 5), new Grid.Position(3, 6),
+          new Grid.Position(3, 7), new Grid.Position(3, 8),
+          new Grid.Position(3, 9)
+        ], // machines_positions
+        // [], // hidden_positions
+        Goals.FIX_ALL_MACHINES // goal
+      ),
+      "9{}",  // initialCode
+      Constants.Lesson03.SUCCESS_MESSAGE,
+      14
```
Comment (gpoesia) [inline src/lesson/lesson03.js]: Is this really 14? I managed to do it with 15.

**Comment by nildo** [comment]
Fixed every observation here. Also added the new sprites and new style for the robot.

**Pull request merged**